config/jobs/kubernetes/sig-instrumentation/sig-instrumentation-kind-periodics.yaml

```diff
@@ -31,7 +33,18 @@ periodics:
       - wrapper.sh
       - bash
       - -c
-      - curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" && e2e-k8s.sh
+      # e2e-k8s.sh has no env variable for injecting feature gates or runtime config,
+      # therefore it gets patched here with sed. This obviously has to make assumptions
+      # about the current implementation. https://github.com/kubernetes-sigs/kind/pull/3023
+      # is one possible solution, but it is unclear if or when something like that will
+      # become available.
+      - >
+        curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" &&
+        sed -i -e "s/feature_gates=.*/feature_gates='{\"DynamicResourceAllocation\":true,\"ContextualLogging\":true}'/"
```

Comment:
@BenTheElder is this reasonable?

Reply:
This is maybe another way of doing it? https://github.com/kubernetes/test-infra/blob/master/experiment/kind-detect-local-e2e.sh#L112-L144

Reply:
That only allows disabling feature gates (everything that isn't GA), but not enable additional ones.